### ISSUE TITLE
Add license

### ISSUE THREAD

**Issue by 1nikolas** [HBiSoft/PickiT#21]
This library needs a license so we can use it in our apps. Have a look at [here](https://choosealicense.com/) to choose the appropriate one (I recommend MIT). Then add a `LICENSE` file in the root of this repo.

According to [this page](https://choosealicense.com/no-permission/):
_When you make a creative work (which includes code), the work is under exclusive copyright by default. Unless you include a license that specifies otherwise, **nobody else can copy, distribute, or modify your work** without being at risk of take-downs, shake-downs, or litigation._

In other words, nobody can legally use this library except @HBiSoft who made it.

**Comment by HBiSoft** [comment]
Thanks, I have added the license.

**Comment by fat-fellow** [comment]
File should be called LICENSE, not License.txt
Some automatic tools cannot find the license
